sonar-ws/src/main/gen-java/org/sonarqube/ws/Rules.java

```diff
@@ -486,12 +441,6 @@ public boolean hasName() {
         }
       }
 
-      private void initFields() {
-        repository_ = "";
-        key_ = "";
-        internalKey_ = "";
-        name_ = "";
-      }
       private byte memoizedIsInitialized = -1;
       public final boolean isInitialized() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) "final" is redundant in this context. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2333)